sphinx/application.py

```diff
@@ -29,7 +28,7 @@
     directives, roles
 
 import sphinx
-from sphinx import package_dir, locale
+from sphinx import package_dir, locale, extensions
```

Comment:
Finally, I renamed it to singular one. At same time, I renamed the functions to `load_extension()` and `verify_required_extensions()`.
Thanks for good review!